scripts/ruby-tasks/visual_recognition_v3.rake

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
```

Comment:
we stopped using tasks for `vis rec`, so we can delete this one, and the patch file.

Reply:
done